Library/Formula/gnome-icon-theme.rb

```diff
@@ -18,12 +15,28 @@ class GnomeIconTheme < Formula
   depends_on "gtk+3" => :build # for gtk3-update-icon-cache
   depends_on "icon-naming-utils" => :build
   depends_on "intltool" => :build
+  depends_on "librsvg" => :build
 
   def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
-                          "--enable-icon-mapping",
```

Comment:
The configure script gave a warning about it being an unsupported option:
`configure: WARNING: unrecognized options: --enable-icon-mapping`